pom.xml

```diff
@@ -1183,7 +1183,7 @@
         <equinox.osgi.version>3.10.2.v20150203-1939</equinox.osgi.version>
         <equinox.osgi.services.version>3.4.0.v20140312-2051</equinox.osgi.services.version>
         <carbon.kernel.version>5.1.0</carbon.kernel.version>
-        <transport.http.version>6.0.94</transport.http.version>
+        <transport.http.version>6.0.97</transport.http.version>
```

Comment:
We can have it as 0.95 otherwise it shall fail because of compilation failure because of keep-alive change.

Reply:
0.95 has a build break..